pkg/system/reconciler.go

```diff
@@ -99,21 +101,23 @@ func NewReconciler(
 ) *Reconciler {
 
 	r := &Reconciler{
-		Request:             req,
-		Client:              client,
-		Scheme:              scheme,
-		Recorder:            recorder,
-		OperatorVersion:     version.Version,
-		CoreVersion:         options.ContainerImageTag,
-		Ctx:                 context.TODO(),
-		Logger:              logrus.WithField("sys", req.Namespace+"/"+req.Name),
-		NooBaa:              util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaa_cr_yaml).(*nbv1.NooBaa),
-		ServiceAccount:      util.KubeObject(bundle.File_deploy_service_account_yaml).(*corev1.ServiceAccount),
-		CoreApp:             util.KubeObject(bundle.File_deploy_internal_statefulset_core_yaml).(*appsv1.StatefulSet),
-		NooBaaDB:            util.KubeObject(bundle.File_deploy_internal_statefulset_db_yaml).(*appsv1.StatefulSet),
+		Request:         req,
+		Client:          client,
+		Scheme:          scheme,
+		Recorder:        recorder,
+		OperatorVersion: version.Version,
+		CoreVersion:     options.ContainerImageTag,
+		Ctx:             context.TODO(),
+		Logger:          logrus.WithField("sys", req.Namespace+"/"+req.Name),
+		NooBaa:          util.KubeObject(bundle.File_deploy_crds_noobaa_io_v1alpha1_noobaa_cr_yaml).(*nbv1.NooBaa),
+		ServiceAccount:  util.KubeObject(bundle.File_deploy_service_account_yaml).(*corev1.ServiceAccount),
+		CoreApp:         util.KubeObject(bundle.File_deploy_internal_statefulset_core_yaml).(*appsv1.StatefulSet),
+		NooBaaDB:        util.KubeObject(bundle.File_deploy_internal_statefulset_db_yaml).(*appsv1.StatefulSet),
+		// NooBaaPostgresDB:    util.KubeObject(bundle.File_deploy_internal_statefulset_postgres_db_yaml).(*appsv1.StatefulSet),
+		ServiceDb: util.KubeObject(bundle.File_deploy_internal_service_db_yaml).(*corev1.Service),
+		// ServicePostgresDb:   util.KubeObject(bundle.File_deploy_internal_service_postgres_db_yaml).(*corev1.Service),
 		ServiceMgmt:         util.KubeObject(bundle.File_deploy_internal_service_mgmt_yaml).(*corev1.Service),
 		ServiceS3:           util.KubeObject(bundle.File_deploy_internal_service_s3_yaml).(*corev1.Service),
-		ServiceDb:           util.KubeObject(bundle.File_deploy_internal_service_db_yaml).(*corev1.Service),
```

Comment:
Are you sure this should be removed, I could find the lines that reconcile the service for the mongodb case

Reply:
It is not being removed, It has been reorder

Reply:
Sorry missed it. I now see it was moved. But in any case it would be better if you have both versions on the reconciler then choose which one to update k8s with